packages/docs/next-doc-site/public/fonts.css

```diff
@@ -1,43 +1,149 @@
+/* Studio-Feixen-Sans Regular */
+
+@font-face {
+  font-family: 'Studio-Feixen-Sans';
+  src: url('/fonts-split/StudioFeixenSansRegular-latin.woff2')
+      format('woff2'),
+    url('/fonts-split/StudioFeixenSansRegular-latin.woff')
+      format('woff');
+  font-display: swap;
+  font-weight: 400;
+  font-style: normal;
+  unicode-range: U+00A1,U+00AA-00AB,U+00AF,U+00B8,U+00BB,U+00BF-00D6,U+00D8-00F6,U+00F8-00FF,U+0131,U+0152-0153,U+02B0-02FF;
+}
+
+@font-face {
+  font-family: 'Studio-Feixen-Sans';
+  src: url('/fonts-split/StudioFeixenSansRegular-latin-extended.woff2')
+      format('woff2'),
+    url('/fonts-split/StudioFeixenSansRegular-latin-extended.woff')
+      format('woff');
+  font-display: swap;
+  font-weight: 400;
+  font-style: normal;
+  unicode-range: U+0100-0130,U+0132-0151,U+0154-017F,U+0180-024F,U+1E00-1EFF;
+}
+
 @font-face {
   font-family: 'Studio-Feixen-Sans';
-  src: url('https://waffles.datacamp.com/fonts/StudioFeixenSans-Regular.woff2')
+  src: url('/fonts-split/StudioFeixenSansRegular-rest.woff2')
       format('woff2'),
-    url('https://waffles.datacamp.com/fonts/StudioFeixenSans-Regular.woff')
+    url('/fonts-split/StudioFeixenSansRegular-rest.woff')
```

Comment:
Ahh maybe we need to use this one instead to accommodate the names

Reply:
For preload it's enough to load only the smallest font subset to prevent possible reflows caused by font asap. After that browser will load other required font subsets (that seems priority is still high, most likely it will start downloading before other assets).